src/react-components/input/SelectInputField.js

```diff
@@ -110,7 +110,7 @@ SelectInputField.propTypes = {
       PropTypes.string,
       PropTypes.number,
       PropTypes.shape({
-        id: PropTypes.string.isRequired,
+        // id: PropTypes.string.isRequired,
```

Comment:
This change fixes a bug in the entry flow. When the microphone options menu is presented, there are no `id`s.